src/components/List/ComplaintCard/ComplaintCard.spec.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import { ComplaintCard } from './ComplaintCard';
+import { IntlProvider } from 'react-intl';
+
+describe('ComplaintCard', () => {
+  let itemFixture;
+  beforeEach(() => {
+    itemFixture = {
+      complaint_what_happened: '',
+      date_sent_to_company: '2022-11-16T12:00:00-05:00',
+      zip_code: '12345',
+      tags: null,
+      has_narrative: false,
+      complaint_id: '7990095',
+      timely: 'Yes',
+      consumer_consent_provided: null,
+      company_response: 'In progress',
+      submitted_via: 'Web',
+      company: 'JP Morgan',
+      date_received: '2022-11-16T12:00:00-05:00',
+      state: 'FL',
+      consumer_disputed: 'N/A',
+      company_public_response: null,
+    };
+  });
+
+  test('it renders correctly', () => {
+    const tree = renderer
+      .create(
+        <IntlProvider locale="en">
+          <ComplaintCard row={itemFixture} />
+        </IntlProvider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
I could add this snapshot in easily enough since the component doesn't depend on the redux store but I didn't for other two tests. Do we still want to be snapshot testing as we upgrade the tests from Enzyme to Testing Library? I can look into how to do it with Redux if we do; just wanted to check in about it.

Reply:
I'm okay with not doing snapshot testing and removing them from CCDB as long as we're testing for the values we need via testing-library.

Reply:
Removed the ComplaintCard snapshot and added additional assertions in 2a656e9